mods/farming/api.lua

```diff
@@ -83,10 +83,30 @@ farming.register_hoe = function(name, def)
 		end
 	})
 	-- Register its recipe
-	minetest.register_craft({
-		output = name:gsub(":", "", 1),
```

Comment:
Instead of these `gsub`s, change the code so that the leading `:` is only added where needed.

If you did strip it after adding it `:sub(2)` would be much faster though.